image_proc/include/edge_aware.h

```diff
@@ -1,7 +1,7 @@
 /*********************************************************************
 * Software License Agreement (BSD License)
 * 
-*  Copyright (c) 2008, Willow Garage, Inc.
+*  Copyright (c) 2019, Andreas Klintberg.
```

Comment:
I believe that the appropriate thing is to append to the copyright holders here.